internal/util/importutil/import_options.go

```diff
@@ -92,3 +93,12 @@ func ParseTSFromOptions(options []*commonpb.KeyValuePair) (uint64, uint64, error
 	}
 	return tsStart, tsEnd, nil
 }
+
+// IsBackup returns if the request is triggered by backup tool
+func IsBackup(options []*commonpb.KeyValuePair) bool {
+	isBackup, err := funcutil.GetAttrByKeyFromRepeatedKV(BackupFlag, options)
+	if err != nil || isBackup != "true" {
```

Comment:
shall be support case insensitivity here?

Reply:
Not necessary, better to have it. Updated